### PR TITLE
[NUI] Add MaskingMode property in imageView to use Rendering Time Masking

### PR DESCRIPTION
**Pull request by bshsqa**
Signed-off-by: seungho <sbsh.baek@samsung.com>

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by jaehyun0cho — APPROVED** [review]
LGTM

File: src/Tizen.NUI/src/public/BaseComponents/ImageView.cs [hunk 1]
```diff
@@ -235,6 +235,24 @@ public enum LoadingStatusType
...
         }
 
+        /// <summary>
+        /// Enumeration for MaskingMode of image.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum MaskingModeType
+        {
+            /// <summary>
+            /// Loading preparing status.
```
Comment (hinohie) [inline src/Tizen.NUI/src/public/BaseComponents/ImageView.cs:245]: I think this comment is copy-paste. We need to write something more good sentence.

Comment (bshsqa) [reply src/Tizen.NUI/src/public/BaseComponents/ImageView.cs]: Thank you I fixned

File: src/Tizen.NUI/src/public/Visuals/VisualConstants.cs [hunk 2]
```diff
@@ -945,6 +945,14 @@ public struct ImageVisualProperty
         /// </remarks>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly int RedrawInScalingDown = NDalic.ImageVisualOrientationCorrection + 11;
+
+        /// <summary>
+        /// @brief Whether to apply mask on the GPU or not.
+        /// @details Name "MaskingMode", type MaskingModeType (Property::INTEGER).
+        /// @note It is used in the ImageVisual, and AnimatedImageVisual.The default is false.
```
Comment (hinohie) [inline src/Tizen.NUI/src/public/Visuals/VisualConstants.cs:952]: Isn't default is `MaskingOnLoading` ? not `false`

**Review by hinohie — APPROVED** [review]
Minor commnet

**Review by hinohie — APPROVED** [review]
LGTM

**Review by taehyub — APPROVED** [review]
LGTM

**Comment by everLEEst** [comment]
can we go now?

**Pull request merged**